variable/energy/energy-final.yaml

```diff
@@ -11,87 +12,94 @@ Final Energy|Electricity:
    unit: EJ/yr
 
 Final Energy|Industry|Electricity:
-   description: Final energy consumption by the industrial sector of
-                electricity (including on-site solar pv), excluding
-                transmission/distribution losses
+   description: Final Electricity consumption excluding transmission/distribution losses,
+                sector - by the industrial sector (including on-site solar pv),  excluding industries that produce energy
    unit: EJ/yr
 
 Final Energy|Industry|Fossil:
-   description: Final energy consumption by the industrial sector of fossil
-                fuels
+   description: Final fossil energy consumption, excluding transmission/distribution losses 
+                source - fossil
+                sector - by the industrial sector , excluding industries that produce energy
    unit: EJ/yr
 
 Final Energy|Industry|Gases:
-   description: Final energy consumption by the industrial sector of gases
-                (natural gas, biogas, coal-gas), excluding
-                transmission/distribution losses
+   description: Final gas energy consumption, excluding transmission/distribution losses,
+                source - (natural gas, biogas, coal-gas) gas
+                sector - by the industrial sector , excluding industries that produce energy
    unit: EJ/yr
 
 Final Energy|Industry|Geothermal:
-   description: Final energy consumption by the industrial sector of geothermal
-                energy
+   description: Final geothermal energy consumption, excluding transmission/distribution losses, 
+                source - (natural gas, biogas, coal-gas) gas
+                sector - by the industrial sector , excluding industries that produce energy
    unit: EJ/yr
 
 Final Energy|Industry|Heat:
-   description: Final energy consumption by the industrial sector of heat
-                (e.g., district heat, process heat, solar heating and warm
-                water), excluding transmission/distribution losses
+   description: Final energy consumption, excluding transmission/distribution losses
+                source - all
+                use - heat (e.g., district heat, process heat, solar heating and warm
+                water)
+                sector - by the industrial sector, excluding industries that produce energy    
    unit: EJ/yr
 
 Final Energy|Industry|Hydrogen:
    description: Final energy consumption by the industrial sector of hydrogen
+   # ambiguous: is it industries that produce hydrogen (and thus need energy to produce it) or that consume hydrogen as an energy source
    unit: EJ/yr
 
 Final Energy|Industry|Liquids:
    description: Final energy consumption by the industrial sector of refined
                 liquids (conventional & unconventional oil, biofuels,
                 coal-to-liquids, gas-to-liquids)
+   # ambiguous: is it industries that produce liquids (and thus need energy to produce it) or that consume liquids as an energy source
```

Comment:
maybe just switching the description to "consumption of refined liquids by the industrial sector" reduces the ambiguity?

Reply:
Yes, agree